EOLib/PacketHandlers/Guild/GuildLeaveHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AutomaticTypeMapper;
+using EOLib.Domain.Interact;
+using EOLib.Domain.Interact.Guild;
+using EOLib.Domain.Login;
+using EOLib.Domain.Map;
+using EOLib.Net.Handlers;
+using Moffat.EndlessOnline.SDK.Protocol.Net;
+using Moffat.EndlessOnline.SDK.Protocol.Net.Server;
+using Optional;
+using System.Collections.Generic;
+
+namespace EOLib.PacketHandlers.Guild
+{
+    [AutoMappedType]
+    public class GuildLeaveHandler : InGameOnlyPacketHandler<GuildKickServerPacket>
```

Comment:
There a few handlers that don't follow this, but can you rename this to `GuildKickHandler` to match the packet id? I've been trying to update handler names to this convention when I see they're out of sync.